### PR TITLE
Add jsdoc comments for printPage api function

### PR DESCRIPTION
**Pull request by jerrylee17**

### CODE REVIEW

File: src/APIFunctions/2DPrinting.js [hunk 1]
```diff
@@ -43,9 +43,16 @@ export function parseRange(pages, maxPages) {
 
 /**
  * Print the page
- * @param {Object} data         Encoded file
- * @returns {PrintApiResponse}  Containing information for if
- *                              the page is printing
+ * @param {Object} data - Encoded file and its configurations
+ * @param {String|undefined} data.raw - Encoded file
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:47]: are you sure the uploaded file is represented as a string?

File: src/APIFunctions/2DPrinting.js [hunk 2]
```diff
@@ -43,9 +43,16 @@ export function parseRange(pages, maxPages) {
...
+ * @param {Number|undefined} data.copies - Number of copies
+ * @param {String|undefined} data.sides - Sides to print:
+ * one-sided or two-sided
+ * @param {String|undefined} data.pageRanges - Pages to print
+ * @param {String|undefined} data.destination - Destination of
+ * the printer
+ * @returns {PrintApiResponse} - Containing information for if
+ * the page is printing
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:55]: it would have printed by now

File: src/APIFunctions/2DPrinting.js [hunk 3]
```diff
@@ -43,9 +43,16 @@ export function parseRange(pages, maxPages) {
...
- * @param {Object} data         Encoded file
- * @returns {PrintApiResponse}  Containing information for if
- *                              the page is printing
+ * @param {Object} data - Encoded file and its configurations
+ * @param {String|undefined} data.raw - Encoded file
+ * @param {Number|undefined} data.copies - Number of copies
+ * @param {String|undefined} data.sides - Sides to print:
+ * one-sided or two-sided
+ * @param {String|undefined} data.pageRanges - Pages to print
+ * @param {String|undefined} data.destination - Destination of
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:52]: i don't think we need this anymore as the rpc server decides, what do you think about removing it

File: src/APIFunctions/2DPrinting.js [hunk 4]
```diff
@@ -43,9 +43,16 @@ export function parseRange(pages, maxPages) {
...
  * Print the page
- * @param {Object} data         Encoded file
- * @returns {PrintApiResponse}  Containing information for if
- *                              the page is printing
+ * @param {Object} data - Encoded file and its configurations
+ * @param {String|undefined} data.raw - Encoded file
+ * @param {Number|undefined} data.copies - Number of copies
+ * @param {String|undefined} data.sides - Sides to print:
+ * one-sided or two-sided
+ * @param {String|undefined} data.pageRanges - Pages to print
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:51]: give an examples of ranges like you did on line 50

File: src/APIFunctions/2DPrinting.js (hunk 1 above)
Comment (jerrylee17) [reply src/APIFunctions/2DPrinting.js:47]: Base64 string

File: src/APIFunctions/2DPrinting.js (hunk 3 above)
Comment (jerrylee17) [reply src/APIFunctions/2DPrinting.js:52]: Me neither, we can remove it

File: src/APIFunctions/2DPrinting.js (hunk 1 above)
Comment (evanugarte) [reply src/APIFunctions/2DPrinting.js:47]: ok cool

File: src/APIFunctions/2DPrinting.js (hunk 3 above)
Comment (evanugarte) [reply src/APIFunctions/2DPrinting.js:52]: remove it in this pr its ok

Comment (jerrylee17) [reply src/APIFunctions/2DPrinting.js:52]: We need to have a separate PR in SCE-RPC to fix the function then. otherwise, it will break

**Comment by jerrylee17** [comment]
This PR will break the RPC printing function. Need a separate PR in SCE-RPC to remove `destination` from sendPrintRequest

**Comment by evanugarte** [comment]
@jerrylee17 open it when you can, ill test both out to make sure they work and we merge

File: src/APIFunctions/2DPrinting.js (hunk 1 above)
Comment (evanugarte) [reply src/APIFunctions/2DPrinting.js:47]: @markbrogan13

**Pull request merged**